modules/dkimVerifier.jsm.js

```diff
@@ -27,15 +27,15 @@
 // options for ESLint
 /* eslint strict: ["warn", "function"] */
 /* global Components, Services */
-/* global Logging, Key, Policy, MsgReader */
-/* global dkimStrings, addrIsInDomain2, domainIsInDomain, stringEndsWith, stringEqual, writeStringToTmpFile, DKIM_SigError, DKIM_TempError, DKIM_Error */
+/* global Logging, Key, Policy, MsgReader, rfcParser */
+/* global dkimStrings, addrIsInDomain2, domainIsInDomain, stringEndsWith, stringEqual, writeStringToTmpFile, toType, DKIM_SigError, DKIM_TempError, DKIM_Error */
 /* exported EXPORTED_SYMBOLS, Verifier */
 
 // @ts-ignore
 const module_version = "2.3.0";
 
 var EXPORTED_SYMBOLS = [
-	"Verifier"
+    "Verifier"
```

Comment:
I completely agree. And actually, I thought I had an addon, which takes care of this and inserts tabs/spaces depending of the rest of the content.
I've converted all files to space indention only.
Can I add some rules to your linter config, which would flag the use of tabs as warning (and ideally also white space at the end of line, which sometimes happens unintentionally).

Reply:
>  I've converted all files to space indention only.

Please revert this. Currently all `*.js` files uses tabs for indention. If you want to change this for the 2.x branch that could be up for discussions. But formatting changes would needed to be in a separate PR instead of mixed together with functional changes, and a general change to all files and not just to a few.

> I thought I had an addon, which takes care of this and inserts tabs/spaces depending of the rest of the content.

The repository contains a [`.editorconfig`](https://editorconfig.org/) file, specifying how white space should be handled, which is also supported by an add-on for VS Code. If you have not installed it yet I would recommend you do.

> Can I add some rules to your linter config, which would flag the use of tabs as warning (and ideally also white space at the end of line, which sometimes happens unintentionally).

There probably are some ways, but at least the standard ESLint is going away from also including such style rules. Maybe a plugin for it exists to include this.
For me the `.editorconfig` file together with the add-on for VS Code was so far enough to ensure the white space was consisted.
I briefly looked and there seems to exist a tool to check this in the CI (https://github.com/editorconfig-checker/editorconfig-checker). But I have not yet used that myself.

Long term I should probably look into adapting one of the "standard" formatter for JavaScript instead of the one build into VS Code. That could ensure that more than just the white space is consistent.

Reply:
Just noticed that I mixed up tabs/spaces in my initial comment, which was probably the reason you converted everything to spaces. Sorry about that,

Reply:
I've changed the files back to tab indention.
I don't want to change it, but I like it consistent.

Reply:
Please have a lock at the diff to the old state. You are now using tabs in a lot of places there before spaces were used.

In general tabs should be only used for indention of code, not for any alignment e.g. in comments. I.e. the files should continue to look "good" and be properly aligned no matter what the tab with is set to. Meaning tabs should only every occur at the beginning of a line.

Reply:
I hope, I got it right now :) 
Sorry, doing something automatically isn't always the best...

Reply:
Yes looks good now.

Note as most of the white space changes there in separate commit you could have just use git to revert it.